### DISCUSSION TITLE
Export xmind

### DISCUSSION THREAD

**Discussion by shadowdevnotreal** [soxoj/maigret#713] [General]
1st = love this tool.
It is by far the gold standard for docier reporting.

I saw your parsing code and had a question maybe you can help me on.
You have one of the only tools I have seen that can successfully export into html, pdf, xmind, json, etc.

I would like to add that xmind & json export code to my own tool.
Can you please guide me on how I can take that code and add it to my OSINT tool........please.

I am a new developer and working on a universal import, export OSINT reporting tool, but cannot figure out how to clip that parsing code to make it work with my tool

**Comment by soxoj** [comment]
Hi, so what the problem use my code for export?

**Reply by shadowdevnotreal** [reply]
No problem at all.
You're one of the only people that has export parse code to any format.

I'm asking, how can I add that export parse code to my own OSINT reporting tool.

I get depreciated warnings and its tailored for maigret.

I'm trying to make a universal export feature to my own code and you're parse code is perfect!!!

I'm a new developer and learn quickly but what lines do I need to replace so I can add it to mine?

**Comment by shadowdevnotreal** [comment]
For better context.
I'm working on a OSINT tool that can import data of any type, visualize it, then export it to any type the user wants.

**Comment by shadowdevnotreal** [comment]
My assumption is if the code can export, it can also import.

Either way..... I'm asking how to modify it to work with anything. 

Right now it only works with maigret, but it's the best code I've seen that has xmind export as well.

Any help would be immensely appreciated and of course I'll credit you as the source! 😁

**Reply by soxoj** [reply]
Well, we should take a look at your code first to understand what do you need to add.

**Reply by shadowdevnotreal** [reply]
Give me a few days and I'll start back on it.
I thought maybe there was a way to just run the xmind export code from the parse code.
I was trying to just run that part of your code.

For clarity, I'm looking to modify your parse export code just didn't know what I needed to add to have it choose a file in the same folder as the script.

Like if I had something called OSINT.pdf I didn't know where to plug that into the code.

**Comment by shadowdevnotreal** [comment]
I'm going to start working on my code again soon.
I respect your dedication and hard work contribute to the OSINT community. 

If it's ok, I'd rather just invite you to the repo.

I'm not ready to release this yet.
Or if I can reach out to you via email or telegram.

**Comment by shadowdevnotreal** [comment]
So.....yeah.
Turns out I reached out to this guy on Telegram.
Him and his friends start talking in Russian about spying on America then ask each other if they want to start trolling my friend for fun game.

What an amazing human this guy is, right.
Your a POS dude.
